packages/@n8n/permissions/src/scope-information.ts

```diff
@@ -0,0 +1,24 @@
+import { RESOURCES } from './constants.ee';
+import type { Scope, ScopeInformation } from './types.ee';
+
+function buildResourceScopes() {
```

Comment:
What would you think of a more 'declarative' way, like this:
```
function buildResourceScopes(): Scope[] {
	const resourceScopes = Object.entries(RESOURCES).flatMap(
		([resource, operations]) => [
			...operations.map((op) => `${resource}:${op}` as const),
			`${resource}:*` as const,
		],
	) as Scope[];

	resourceScopes.push('*' as const); // Global wildcard
	return resourceScopes;
}
```

Maybe a bit less performant though